.github/workflows/update_formula.yml

```diff
@@ -0,0 +1,46 @@
+name: Homebrew update formula
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        required: true
+        description: "fp release version (just numbers without the v, e.g.: 2.2.0)"
+        type: string
+      sha_aarch64_apple:
+        required: true
+        description: "SHA256 for macOS arm64 binary"
+        type: string
+      sha_x86_64_apple:
+        required: true
+        description: "SHA256 for macOS Intel binary"
+        type: string
+      sha_x86_64_linux:
+        required: true
+        description: "SHA256 for Linux Intel binary"
+        type: string
+    secrets:
+      GITHUB_TOKEN:
+        required: true
+
+jobs:
+  homebrew:
+    name: Update Homebrew formula
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Run update script
+        run: ./update_homebrew_formula.sh
+        shell: bash
+
+      - name: Commit changed file
+        env:
+          COMMIT_AUTHOR: Author of the commit
+          COMMIT_MESSAGE: Message of the commit
+        run: |
+          git config --global user.name "${{ env.COMMIT_AUTHOR }}"
+          git config --global user.email "username@users.noreply.github.com"
+          git commit -a -m "${{ env.COMMIT_MESSAGE }}"
```

Comment:
Maybe it is safer to do a separate `git add ...` and `git commit ...` with `-a`.